### DISCUSSION TITLE
Mentalhealth-心理健康大模型

### DISCUSSION THREAD

**Discussion by xiexiaoshinick** [InternLM/Tutorial#333] [project]
参考[qiuhuachuan/smile: SMILE: Single-turn to Multi-turn Inclusive Language Expansion via ChatGPT for Mental Health Support paper: (github.com)](https://github.com/qiuhuachuan/smile)
中文心理健康支持模型由internlm_7b 监督指令微调得到。数据集采用Smile开源的真实的心理互助 QA为多轮的心理健康支持多轮对话，提高了通用语言大模型在心理健康支持领域的表现，更加符合在长程多轮对话的应用场景。

**Comment by MING-ZCH** [comment]
可以的，看好~

**Comment by TAICHIKF** [comment]
挺不错

**Comment by ZixinxinWang** [comment]
举手报名！

**Comment by SiliconSisyphus** [comment]
你好，队伍人满了吗

**Comment by BarryYin** [comment]
想加入，有位置吗

**Comment by TAICHIKF** [comment]
@xiexiaoshinick 大佬，和大家一起做吗

**Comment by leonfrank** [comment]
还有位置吗

**Comment by vansin** [comment]
@xiexiaoshinick 请和浦语小助手取得联系哟~ （微信号：InternLM）

**Comment by leonfrank** [comment]
求加好友，微信号fyl19941230
